test/unit-data/ila_sim_test/compute_external_mem.cc

```diff
@@ -1,76 +1,72 @@
 #include "systemc.h"
-#include "test.h"
+#include "TEST.h"
 void TEST::compute() {
-  if (decode_98909_update_TEST_XRAM_iter < decode_98909_update_TEST_XRAM_map.size()) {
+  if (decode_1827_update_TEST_XRAM_iter < decode_1827_update_TEST_XRAM_map.size()) {
```

Comment:
Auto-generated code should not be checked in.

Reply:
This is for testing purpose